docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,634 @@
+| `service_component_device_orientation_changed_cb` | Event type for the callback function that is responsible for changing the display orientation to match the device orientation. |
+| `service_component_language_changed_cb`           | Event type for the callback function that is responsible for refreshing the display into a new language. |
+| `service_component_region_format_changed_cb`      | Event type for the callback function that is responsible for refreshing the display into a new time zone. |
+| `service_component_suspended_state_changed_cb`    | Event type for the callback function that is responsible for taking necessary actions before entering the suspended state or after exiting from the state. |
+
+
+
+<a name="attribute"></a>
+## Application Attributes
+
+Define your component-based application attributes in the manifest file. The attributes determine the application behavior. The following code example illustrates how you can define the attributes:
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="5.5" package="@PACKAGE_NAME@" install-location="internal-only" version="0.1.1">
+    <label>Sample</label>
+    <author email="tizenappfw@samsung.com" href="www.samsung.com">Tizen App Framework</author>
+    <description>Sample</description>
+    <component-based-application appid="org.tizen.base-component" exec="@BINDIR@/base-component" type="capp">
+        <label>Base-component application</label>
+        <icon>@DESKTOP_ICON@</icon>
+        <service-component id="base-service" main="false">
+            <label>base-service</label>
+        </service-component>
+        <frame-component id="base-frame" launch_mode="caller" main="true" icon-display="true" taskmanage="true">
+            <icon>org.tizen.sample.png</icon>
+            <label>base-frame</label>
+            <label xml:lang="en-us">base-frame</label>
+            <label xml:lang="ko-kr">base-frame[KOR]</label>
+        </frame-component>
+    </component-based-application>
+    <privileges>
+        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+    </privileges>
+</manifest>
+```
+
+Following are the main attributes:
+
+- `id`
+
+  Every component element has an `id` attribute. The attribute ID must be globally unique in a device. If not, the application installation fails. This attribute is also used by other applications to confirm which component must be created.
+
+- `main`
+
+  Component-based applications can have multiple components and there is one main component. The main component is the component that is created when the component-based application receives a launch request without a specific component ID. To set the main component, set the `main` attribute to `true`.
+
+- `icon-display`
+
+  This attribute is only for the frame-component element. If this value is `true`, the home application will display an icon on the app tray for this frame-component.
```

Comment:
In the second part of the sentence, can we rephrase ase:
If this value is set to `true`, the home application will display an icon on the app tray for the frame-components.

Please verify and share your opinion so that we can rephrase.